### PR TITLE
[fix](stats) Fix data size calculation of auto sample

### PR DESCRIPTION
**Pull request by Kikyou1997**
1. Fix data size calculation of auto sample, before this pr, the data size is include all the replicas
2. Move some auto analyze related options to global session variable
3. Add some logs

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Review by Jibing-Li — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java [hunk 1]
```diff
@@ -1568,14 +1568,18 @@ public Partition replacePartition(Partition newPartition) {
         return oldPartition;
     }
 
-    public long getDataSize() {
+    public long getDataSize(boolean singleReplica) {
         long dataSize = 0;
         for (Partition partition : getAllPartitions()) {
-            dataSize += partition.getDataSize(false);
+            dataSize += partition.getDataSize(singleReplica);
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:1574]: for numberLike/dateLike column, it is not necessary to compute dateSize. 
for example for int column
dataSize = 4* rowCount

File: docs/zh-CN/docs/query-acceleration/statistics.md [hunk 2]
```diff
@@ -249,18 +249,18 @@ SHOW AUTO ANALYZE [ptable_name]
 |full_auto_analyze_end_time|自动统计信息收集结束时间|02:00:00|
 |enable_auto_sample|是否开启大表自动sample，开启后对于大小超过huge_table_lower_bound_size_in_bytes会自动通过采样收集| false|
 |auto_analyze_job_record_count|控制统计信息的自动触发作业执行记录的持久化行数|20000|
-|huge_table_default_sample_rows|定义开启开启大表自动sample后，对大表的采样行数|200000|
+|huge_table_default_sample_percent|定义开启开启大表自动sample后，对大表的采样比例|10|
```
[the remark this answers is not in the document]
Comment (Kikyou1997) [reply docs/zh-CN/docs/query-acceleration/statistics.md]: 理由是？

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java (hunk 1 above)
Comment (Kikyou1997) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:1574]: 呃，你说的对 但是我没懂这和我这个改动有啥关系

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java (hunk 1 above)
Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:1574]: 在这里判断数据类型，如果是定长字段，就不需要在这里for 循环累加

**Review by englefly — APPROVED** [review]

**Pull request merged**